overcast/runner/__init__.py

```diff
@@ -128,6 +128,99 @@ def get_creds_from_env():
 #    d['cacert'] = os.environ.get('OS_CACERT', None)
     return d
 
+
+class Node(object):
+
+    def __init__(self, name, info, keypair, userdata, runner, record_resource=None):
+        self.record_resource = lambda *args, **kwargs: None
+        self.name      = name
+        self.info      = info
+        self.runner    = runner
+        self.keypair   = keypair
+        self.userdata  = userdata
+        self.server_id = None
+        self.fip_ids   = set()
+        self.port_ids  = set()
+        self.fip_address = None
+        self.server_status = None
+        self.nc = self.runner.get_nova_client()
+        if self.info['image'] in self.runner.mappings.get('images', {}):
+            self.info['image'] = self.runner.mappings['images'][self.info['image']]
+
+        if self.info['flavor'] in self.runner.mappings.get('flavors', {}):
+            self.info['flavor'] = self.runner.mappings['flavors'][self.info['flavor']]
+        self.image = self.nc.images.get(self.info['image'])
+        self.flavor = self.nc.flavors.get(self.info['flavor'])
+
+    ##
+    # This one poll the server and return server status.
+    ##
```

Comment:
Put comments inside the method as a string. Like so:

def poll(...):
    """
    My comment about what this method does
    """

This makes the comment available programmatically to autodoc systems, for instance.